src/test/mempool_tests.cpp

```diff
@@ -547,7 +547,7 @@ BOOST_AUTO_TEST_CASE(MempoolSizeLimitTest)
         AddToMempool(pool, entry.Fee(100LL).FromTx(tx5));
     AddToMempool(pool, entry.Fee(900LL).FromTx(tx7));
 
-    pool.TrimToSize(pool.DynamicMemoryUsage() / 2); // should maximize mempool size by only removing 5/7
+    pool.TrimToSize(pool.DynamicMemoryUsage() * 0.75); // should maximize mempool size by only removing 5/7
```

Comment:
In "Do not allow mempool clusters to exceed configured limits" 5a388c0d595b2318fea4b1dce977e2d5ff1abc48

I try to make it dynamic by without the magic number by trimming using

`pool.DynamicMemoryUsage() - usage_of_5 - usage_of_7` but tests fail because tx 6 is also evicted.
However when subtract only the usage of 5 tests succeed.

I grab the usages by getting the delta in memory usage after insertion of the tx.